lib/helper/REST.js

```diff
@@ -75,12 +76,18 @@ class REST extends Helper {
    * @param {*} request
    */
   async _executeRequest(request) {
+    const _debugRequest = { ...request };
     axios.defaults.timeout = request.timeout || this.options.timeout;
 
     if (this.headers && this.headers.auth) {
       request.auth = this.headers.auth;
     }
 
+    if (request.data instanceof Secret) {
+      _debugRequest.data = '*****';
+      request.data = { ...request.data.toString() };
```

Comment:
I'm confused at this line about what's going on, how come the spread of toString as the request data?  Is .tostring for a secret somehow an object?

Reply:
yeah @Georgegriff `.toString()` is from the `Secret` class.